docs/tutorials/kafka.ipynb

```diff
@@ -0,0 +1,821 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "emslB2EGQMCR"
+      },
+      "outputs": [],
+      "source": [
+        "!curl -sSOL https://archive.ics.uci.edu/ml/machine-learning-databases/00279/SUSY.csv.gz\n",
+        "!gzip -d SUSY.csv.gz"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "4CfKVmCvwcL7"
+      },
+      "source": [
+        "#### Explore the dataset"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "18aR_MsOKToc"
+      },
+      "source": [
+        "The first column is the class label (1 for signal, 0 for background), followed by the 18 features (8 low-level features then 10 high-level features).\n",
+        "\n",
+        "**NOTE:** The first 8 features are kinematic properties measured by the particle detectors in the accelerator. The last ten features are functions of the first 8 features; these are high-level features derived by physicists to help discriminate between the two classes"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "XkXyocIdKRSB"
+      },
+      "outputs": [],
+      "source": [
+        "COLUMNS = [\n",
+        "          #  labels\n",
+        "           'class',\n",
+        "          #  low-level features\n",
+        "           'lepton_1_pT',\n",
+        "           'lepton_1_eta',\n",
+        "           'lepton_1_phi',\n",
+        "           'lepton_2_pT',\n",
+        "           'lepton_2_eta',\n",
+        "           'lepton_2_phi',\n",
+        "           'missing_energy_magnitude',\n",
+        "           'missing_energy_phi',\n",
+        "          #  high-level derived features\n",
+        "           'MET_rel',\n",
+        "           'axial_MET',\n",
+        "           'M_R',\n",
+        "           'M_TR_2',\n",
+        "           'R',\n",
+        "           'MT2',\n",
+        "           'S_R',\n",
+        "           'M_Delta_R',\n",
+        "           'dPhi_r_b',\n",
+        "           'cos(theta_r1)'\n",
+        "           ]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "nC-yt_c9u0sH"
+      },
+      "outputs": [],
+      "source": [
+        "susy_df = pd.read_csv('SUSY.csv', header=None)\n",
```

Comment:
Optional:

-You can read the csv right out of the gzip file, and skip the unzip step.
-Instead of loading the whole thing and throwing away most of it, you could just load one chunk:


```
susy_iterator = pd.read_csv('SUSY.csv.gz',
                            header=None, names=COLUMNS,
                            # setting chunksize returins an an iterator
                            chunksize=100000)
susy_df = next(susy_iterator)
susy_df.head()
```